docs/source/common/lightning_module.rst

```diff
@@ -279,11 +279,16 @@ In this case, implement the `training_step_end` method
          return {'loss': loss, 'pred': pred}
 
      def training_step_end(self, batch_parts):
-         gpu_0_prediction = batch_parts[0]['pred']
-         gpu_1_prediction = batch_parts[1]['pred']
+         # predictions from each GPU
+         predictions = batch_parts['pred]
+         # losses from each GPU
+         losses = batch_parts['loss]
```

Comment:
```suggestion
         predictions = batch_parts['pred']
         # losses from each GPU
         losses = batch_parts['loss']
```

Reply:
Thank you for pointing out, my bad 🤦‍♂️I had resolved this with a new commit